index.js

```diff
@@ -1,7 +1,21 @@
 // Slider Code
-const mySliders=document.querySelectorAll('.slider');
-            mySliders.forEach(mySlider=>mySlider.addEventListener('input',()=>{
-                let valPercent=(mySlider.value/mySlider.max)*100;
-                mySlider.style.background=`linear-gradient(to right,var(--secondary-300) ${valPercent}%,var(--primary-200) ${valPercent}%)`
-            }))
+const mySliders = document.querySelectorAll('.slider');
+mySliders.forEach(mySlider => mySlider.addEventListener('input', () => {
+    let valPercent = (mySlider.value / mySlider.max) * 100;
+    mySlider.style.background = `linear-gradient(to right,var(--secondary-300) ${valPercent}%,var(--primary-200) ${valPercent}%)`
+}))
+
+//Terms and condition dialogue box.
+let termsInfo = document.querySelector('.terms-info');
+termsInfo.addEventListener('click', () => {
+    document.querySelector('.signup-page .login-dialogue .dialogue-container').style.display = "grid";
+})
+document.querySelector('.tertiary.border-between-button').addEventListener('click', () => {
```

Comment:
Keep the elements in a variable first and then add the event listener. That will keep the code clean

Reply:
Done. I unnecessarily used extra variables. Thanks for pointing it out, I have trimmed the fat.